pom.xml

```diff
@@ -114,7 +114,7 @@
     <grpc.version>1.34.1</grpc.version>
     <protobuf.version>3.12.0</protobuf.version>
     <!-- prefer grpc's version of netty -->
-    <netty.version>4.1.71.Final</netty.version>
+    <netty.version>4.1.86.final</netty.version>
```

Comment:
The comment above this leads me to believe we might run into issues if we do not also upgrade the version of grpc.